src/test/java/study/step4/CarNameTest.java

```diff
@@ -0,0 +1,42 @@
+package study.step4;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import study.domain.racingGame.CarNames;
+
+public class CarNameTest {
+
+  @DisplayName("생성 태스트")
+  @Test
+  public void createNameTest() {
+    String[] name = {"a", "b", "c"};
+    assertThat(new CarNames(name)).isInstanceOf(CarNames.class);
+  }
+
+  @DisplayName(" null이거나 비어있는 이름으로 생성 테스트")
+  @ParameterizedTest
```

Comment:
다음과 같이 테스트 설명을 잘성할 수 있습니다 😄 
```suggestion
  @ParameterizedTest(name = "null이거나 비어있는 이름으로 생성 테스트")
```